crates/ratchet-core/src/kernels.rs

```diff
@@ -518,6 +521,7 @@ pub fn kernels() -> &'static HashMap<&'static str, &'static str> {
         ("cache_scalar", include_str!(r"../kernel-handwritten/cache_scalar.wgsl")),
         ("conv_scalar", include_str!(r"../kernel-handwritten/conv_scalar.wgsl")),
         ("f32_index_select_scalar", include_str!(r"../kernel-handwritten/f32_index_select_scalar.wgsl")),
+        ("groupnorm_scalar", include_str!(r"../kernel-handwritten/groupnorm_scalar.wgsl")),
```

Comment:
Did you commit the kernel?

Reply:
I first started with a manual impl before figuring out i can reuse layernorm. I’ll clean this up